pkg/config/config_factory.go

```diff
@@ -32,8 +32,8 @@ func GetConfigProvider() service.ConfigProvider {
 	loc := getConfigFlag()
 	if configContent, ok := os.LookupEnv(envKey); ok {
 		log.Printf("Reading AOT config from environment: %v\n", configContent)
-		loc = "env:" + envKey
+		loc = append(loc, "env:"+envKey)
```

Comment:
```suggestion
		loc = []string{"env:"+envKey}
```

This should replace any value coming from the CLI flag for now.  This ensures compatibility with the prior behavior.  Now that we have the ability to merge multiple configurations we should work out how to transition to allowing supplemental configuration through the environment, but for now let's not break expectations.